control-plane/tests/test_service_persistence.py

```
@@ -3288,6 +3288,45 @@ def test_service_restates_and_deduplicates_live_wazuh_github_audit_ingest_with_c
             (promoted_case.case_id, promoted_case.case_id),
         )
 
+    def test_service_exposes_live_wazuh_entra_id_ingest_through_phase19_operator_surface(
+        self,
+    ) -> None:
+        store, _ = make_store()
+        service = AegisOpsControlPlaneService(
+            RuntimeConfig(
+                postgres_dsn="postgresql://control-plane.local/aegisops",
+                wazuh_ingest_shared_secret="reviewed-shared-secret",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "wazuh_ingest_shared_secret"

(S106)